notebooks/getting_started/bq_dataframes_template.ipynb

```diff
@@ -0,0 +1,1848 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    species_counts = df["species"].value_counts()
</pre><p> </p><p>I worry about this particular integration. <code>to_pandas()</code> would be a bit safer, but I suppose this notebook could serve as a test to discover if matplotlib changes something that would break this integration. What do you think? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/1200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>